### PR TITLE
New new release strings (EXPOSUREAPP-13566)

### PR DESCRIPTION
**Pull request by SamuraiKek**
[One more update to the relase strings](https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.25?node-id=53672%3A36843)

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 1]
```diff
@@ -9,26 +9,30 @@
...
     <string name="new_release_bottom">Änderungen zum Release finden Sie in den App-Informationen unter dem Menüpunkt „Neue Funktionen”.</string>
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
+        <item>Verhaltensempfehlungen aktualisieren</item>
     </string-array>
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
+        <item>Die Verhaltensempfehlungen bei  erhöhten Risiko wurden aktualisiert. Das betrifft unter anderem Empfehlungen zur Reduktion von Kontakten oder dem Tragen von Masken. Angepasst wurden auch die Empfehlungen bei niedrigem Risiko.</item>
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:25]: there is an extra space

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 2]
```diff
@@ -9,26 +9,30 @@
...
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
+        <item>Verhaltensempfehlungen aktualisieren</item>
     </string-array>
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
+        <item>Die Verhaltensempfehlungen bei erhöhten Risiko wurden aktualisiert. Das betrifft unter anderem Empfehlungen zur Reduktion von Kontakten oder dem Tragen von Masken. Angepasst wurden auch die Empfehlungen bei niedrigem Risiko.</item>
     </string-array>
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:26]: ....bei erhöhte**m** Risiko. ... would be correct German, however it is wrong in Figma

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 3]
```diff
@@ -9,26 +9,30 @@
...
+    <string name="release_info_version_body">Mit diesem Update stellen wir Ihnen neben Fehlerbehebungen auch neue und erweiterte Funktionen zur Verfügung.</string>
     <!-- XBUT: Continue button for the release info screen -->
     <string name="release_info_continue_button">Weiter</string>
     <!-- XTXT: New release info footer -->
     <string name="new_release_bottom">Änderungen zum Release finden Sie in den App-Informationen unter dem Menüpunkt „Neue Funktionen”.</string>
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
+        <item>Verhaltensempfehlungen aktualisieren</item>
     </string-array>
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:21]: ... aktaulisiert.  would make more sense to me, but up to the people who create texts

**Review by chiljamgossow — DISMISSED** [review]

**Review by OLVI5 — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml (hunk 2 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:26]: Changed

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml (hunk 3 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:21]: Changed

**Review by chiljamgossow — APPROVED** [review]

**Review by OLVI5 — APPROVED** [review]

**Pull request merged**